### PR TITLE
Discord proxy

### PR DESCRIPTION
**Pull request by awensaunders**
This is the rapidpro side of the changes for issue #1252. This adds the URN scheme, channel type/claim view, and associated tests.

Still to do:
- Figure out how to get a discord icon into the icon font
- Write an appropriate blurb linking to the proxy project once the repo is public

Any feedback would be appreciated.

Also see the changes to [courier](https://github.com/nyaruka/courier/pull/327) and [gocommon](https://github.com/nyaruka/gocommon/pull/28)

### CODE REVIEW

**Comment by awensaunders** [comment]
And I need to resolve the conflicts.

File: temba/channels/types/discord/type.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+from temba.contacts.models import DISCORD_SCHEME
+
+from ...models import ChannelType
+from .views import ClaimView
+
+
+class DiscordType(ChannelType):
+    """
+    A Telegram bot channel
```
Comment (IanLawrence) [inline temba/channels/types/discord/type.py]: This should be Discord not Telegram

Comment (awensaunders) [reply temba/channels/types/discord/type.py]: This is on a pretty old commit. The changeset could be squashed if we want to keep history clean.

**Comment by awensaunders** [comment]
Merged master back in, conflicts resolved.

File: temba/channels/types/discord/type.py (hunk 1 above)
Comment (IanLawrence) [reply temba/channels/types/discord/type.py]: I think we should be careful when commit squashing (and rebasing). I prefer to keep it honest but historically messy so as to not lose critical history.

**Comment by IanLawrence** [comment]
Is there any reason why this has not been merged @nicpottier @rowanseymour ?
Let us know if there is something we should do. Thanks!

**Comment by rowanseymour** [comment]
Hi @IanLawrence sorry sometimes we need need a little prodding to get things merged, but it does seem there's still a bit of work todo on this PR like..

https://github.com/rapidpro/rapidpro/blob/5f77b544dd2edd03f67bd28872230a29408bfeb4/temba/channels/types/discord/type.py#L23-L24

**Comment by awensaunders** [comment]
I've merged main back in, and written a short blurb describing what the discord channel type is. I've also added the discord icon using icomoon to static/fonts/temba.

**Comment by rowanseymour** [comment]
We're currently in code freeze for 6.2 but this looks good to go first thing next week!

**Pull request merged**